packages/PoppyPrint-Tests.package/PPFormatterTest.class/instance/testStandaredWindowWidth1.st

```diff
@@ -0,0 +1,13 @@
+tests - WindowWidth
+testStandaredWindowWidth1
```

Comment:
Should be Standard or could be called Default, maybe Default is a better naming convention